docs/README.md

```diff
@@ -1,11 +1,11 @@
 <!-- prettier-ignore start -->
 <!--
 ---
-title: "Overview of Tekton Triggers"
-linkTitle: "Overview"
+title: "Triggers and EventListeners"
+linkTitle: "Triggers and EventListeners"
 weight: 3
 description: >
-  Tekton Triggers
+  Overview of Tekton Triggers
```

Comment:
How about - `Event Based Triggers for Tekton Pipelines`

Reply:
Fine by me!